src/insert-extension.js

```diff
@@ -34,14 +34,25 @@ export function resetExtensionScriptInsertedOrPresentForTesting() {
 /**
  * Check script info in HTML head and make update if necessary
  * @param {!Window} win
- * @param {!Element} element
  * @param {string} extension
+ * @param {boolean} opt_ignoreElementExistenceCheck if true, will not check and
+ *    require that extension element exist in document
  */
-export function insertAmpExtensionScript(win, element, extension) {
+export function insertAmpExtensionScript(
+    win, extension, opt_ignoreElementExistenceCheck) {
   if (extension == 'amp-embed') {
     extension = 'amp-ad';
   }
-  if (isAmpExtensionScriptRequired(win, element, extension)) {
+  if (!opt_ignoreElementExistenceCheck) {
```

Comment:
I can't think of a situation where this would save us. If we're calling `insertAmpExtensionScript`, it means we definitely have one of those elements on the page, right?